src/components/ButtonAppBar/index.js

```diff
@@ -49,10 +49,16 @@ const divStyle = {
 export function ButtonAppBar() {
   const { userInfo, logout } = useUserContext()
 
+  const history = useHistory()
+
   const signOut = () => {
     logout()
   }
 
+  const Homedir = event => {
```

Comment:
```suggestion
  const handleHomeClick = event => {
```
The name `Homedir` looks like a UI component. If it is a function, we would want that to be like a function.

Reply:
I have changed the name